sunpy/map/header_helper.py

```diff
@@ -49,8 +49,8 @@ def make_fitswcs_header(data, coordinate,
 
     Parameters
     ----------
-    data : `~numpy.ndarray`
-        Array data of Map for which a header is required.
+    data : `~numpy.ndarray` or `tuple`
+        Array data of Map for which a header is required, or the shape of an array.
```

Comment:
should we be more explicit here to the user about the shape - i.e. that (input1, input2) is (y_size, x_size) of array - I know that this is the way it should be as row major in Python but people might just get confused when thinking in terms of a map?